src/parser/jobs/sam/modules/Buffs.js

```diff
@@ -25,7 +25,7 @@ export default class Buffs extends Module {
 	_onComplete() {
 		this.checklist.add(new Rule({
 			name: <Trans id="sam.buffs.checklist.name"> Keep Shifu and Jinpu up </Trans>,
-			description: <Trans id= "sam.buffs.description"> <ActionLink {...ACTIONS.JINPU} /> increases your damage by 10% and <ActionLink {...ACTIONS.SHIFU} /> increases your speed by 10%. Both buffs are key part of Samurai's damage.</Trans>,
+			description: <Trans id= "sam.buffs.description"> <ActionLink {...ACTIONS.JINPU} /> increases your damage by 12% and <ActionLink {...ACTIONS.SHIFU} /> increases your speed by 12%. Both buffs are key part of Samurai's damage.</Trans>,
```

Comment:
nit; Recognise it's just some numeric changes - but given both are 12%, could probably combine the two statements to make it flow a bit better.